README.md

```diff
@@ -97,6 +97,26 @@ programs, even if they have type errors:
 [statically typed parts]: http://mypy.readthedocs.io/en/latest/basics.html#function-signatures
 
 
+IDE & Linter Integrations
+-------------------------
+
+Mypy can be integrated into popular IDEs:
+
+* Vim: [vim-mypy](https://github.com/Integralist/vim-mypy)
+* Emacs: using [Flycheck](https://github.com/flycheck/) and [Flycheck-mypy](https://github.com/lbolla/emacs-flycheck-mypy/issues)
+* Sublime Text: [SublimeLinter-contrib-mypy]
+* Atom: [linter-mypy](https://atom.io/packages/linter-mypy)
+* PyCharm: PyCharm has its own implementation of PEP 484: [PyCharm-PEP484]
+
+[SublimeLinter-contrib-mypy]: (https://packagecontrol.io/packages/SublimeLinter-contrib-mypy)
```

Comment:
Hmm, these links aren't working in preview... ah, aha, it's that there shouldn't be parens.